client/systems.go

```diff
@@ -140,6 +140,28 @@ func (client *Client) RebootToSystem(systemLabel, mode string) error {
 	return nil
 }
 
+type StorageEncryption struct {
+	// A string that can contain:
```

Comment:
not sure why you left out my bit that started with // Support ...

Reply:
Silly me, copy/paste error, I fixed this now.